sycl/plugins/unified_runtime/ur/adapters/cuda/image.cpp

```diff
@@ -131,8 +131,7 @@ cudaToUrImageChannelFormat(CUarray_format cuda_format,
   }
 }
 
-ur_result_t urTextureCreate(ur_context_handle_t hContext,
```

Comment:
This changes the signature of the function, is that intentional?

Reply:
Yes, this is just an internal helper function and we don't need the `hContext` parameter.